apps/sentry/src/connectors/users.test.ts

```diff
@@ -1,65 +1,108 @@
+    const result = await getUsers(validToken, sourceOrganizationId, null);
+    expect(result.members).toEqual(sentryUsers);
+  });
+
+  test('should throw SentryError when token is invalid', async () => {
+    try {
+      await getUsers('invalidToken', sourceOrganizationId, null);
+    } catch (error) {
+      expect((error as SentryError).message).toEqual('Could not retrieve Sentry organization members');
+    }
+  });
+
+  test('should return nextCursor as null when end of list is reached', async () => {
+    const result = await getUsers(validToken, sourceOrganizationId, "last-cursor");
+    expect(result.pagination.nextCursor).toBeNull();
+  });
+
+  test('should return nextCursor when there is next cursor available', async () => {
+    const result = await getUsers(validToken, sourceOrganizationId, 'first-cursor');
+    expect(result.pagination.nextCursor).toEqual('next-cursor');
+  });
+});
+describe('deleteUser', () => {
+  beforeEach(() => {
+    server.use(
+      http.delete(
+        `https://api.sentry.io/api/0/organizations/${sourceOrganizationId}/members/${memberId}`,
+        ({ request }) => {
           if (request.headers.get('Authorization') !== `Bearer ${validToken}`) {
             return new Response(undefined, { status: 401 });
           }
-          const url = new URL(request.url);
-          const pageParam = url.searchParams.get('page');
-          const page = pageParam ? Number(pageParam) : 0;
-          if (page === maxPage) {
-            return Response.json({ nextPage: null, users });
-          }
-          return Response.json({ nextPage: page + 1, users });
-        })
-      );
-    });
-
-    test('should return users and nextPage when the token is valid and their is another page', async () => {
-      await expect(getUsers(validToken, 0)).resolves.toStrictEqual({
-        users,
-        nextPage: 1,
-      });
-    });
+          return new Response(undefined, { status: 204 }); // Assuming a 204 No Content response for successful deletion
+        }
+      )
+    );
+  });
 
-    test('should return users and no nextPage when the token is valid and their is no other page', async () => {
-      await expect(getUsers(validToken, maxPage)).resolves.toStrictEqual({
-        users,
-        nextPage: null,
-      });
-    });
+  test('should delete user successfully when token is valid', async () => {
+    await expect(deleteUser(validToken, sourceOrganizationId, memberId)).resolves.not.toThrow();
+  });
 
-    test('should throws when the token is invalid', async () => {
-      await expect(getUsers('foo-bar', 0)).rejects.toBeInstanceOf(MySaasError);
-    });
+  test('should throw SentryError when token is invalid', async () => {
+    try {
+      await deleteUser("invalidToken", sourceOrganizationId, memberId);
+    } catch (error) {
+      expect(error instanceof SentryError).toBe(true);
+      expect(error.message).toEqual('Could not delete Sentry user');
```

Comment:
single line would be enough:
expect((error as SentryError).message).toEqual('Could not delete Sentry user');

Reply:
Done